Include rule sets in linter rule descriptions (#3498)
dart-lang/site-www: src/tools/linter-rules.md

consider starting with a linter rule set,
which the following packages provide:

<a id="lints"></a>
[lints][]
: Contains two rule sets curated by the Dart team. 
  We recommend using at least the `core` rule set, 
  which is used when [scoring]({{site.pub}}/help/scoring) 
  packages uploaded to [pub.dev]({{site.pub}}). 
  Or, better yet, use the `recommended` rule set, 
  a superset of `core` that identifies additional issues
  and enforces style and format. 
  If you’re writing Flutter code, 
  use the rule set in the [`flutter_lints`](#flutter_lints) package,
  which builds on `lints`.

<a id="flutter_lints"></a>
[flutter_lints][]

> : Contains the `flutter` set of rules
>   that the Flutter team encourages you to use

Reviewer: ```suggestion
: Contains the `flutter` rule set,
  which the Flutter team encourages you to use
```
